.github/workflows/PR-build.yml

```diff
@@ -54,6 +54,13 @@ jobs:
         if: needs.changes.outputs.lint == 'true'
         uses: actions/checkout@v3
 
+      - name: Cache if success
+        id: cache_if_success
+        uses: actions/cache@v3
+        with:
+          path: go.mod
+          key: v1-go-pkg-mod-${{ hashFiles('**/go.sum') }}
```

Comment:
This caches to build. So as long as the go.sum does not change it will keep the downloaded mods. I looked into the set up go and it does this by default. This can be removed.

Reply:
This is running cache on the lint step though so I don't think I get it

Reply:
Removed